src/app/common/types/calendar-type-display-first.ts

```diff
@@ -0,0 +1 @@
+export type CalendarTypeDisplayFirst = 'day' | 'month';
```

Comment:
I think that we should support also a year here. WDYT?

Reply:
Also, what do you think of changing it to enum?

Reply:
Yes, that’s a good idea, but do we have a separate calendar for years?
As I understand it, you want to make enum, and added a year there. But this feature is designed to determine which calendar will open first if the mode is 'day' or 'daytime'. And due to the fact that we do not have a calendar for a separate selection of the year, we cannot define 'year' in the enum.
Let me know if I am wrong.

Reply:
You are right